java-manta-client/src/test/resources/testng.xml

```diff
@@ -40,7 +40,11 @@
     </test>
     <test name="Multipart Tests">
         <classes>
-            <class name="com.joyent.manta.client.multipart.MantaMultipartUploadManagerTest" />
+            <class name="com.joyent.manta.client.multipart.EncryptedMultipartManagerTest" />
```

Comment:
I know this is the stupidest reason in the world. For some reason when I wildcard, it will pick up integration tests when I run the unit tests in IntelliJ and I haven't got around to root causing it.